cli/command/cli.go

```diff
@@ -71,6 +75,9 @@ type DockerCli struct {
 	clientInfo            ClientInfo
 	contentTrust          bool
 	newContainerizeClient func(string) (clitypes.ContainerizedClient, error)
+	contextStore          store.Store
+	currentContext        string
+	dockerEndpoint        docker.Endpoint
```

Comment:
Is this really needed? I don't see where it is used?

Reply:
for now, it is not. But at some point, we want to have a subcommand `docker context create --from-env` which uses whatever host/tls env variables have been set to resolve this endpoint, and put them in a context.

Reply:
Yep but that can be added in this follow up I guess, for now it's like dead code 😉